api/routes/user/controllers.js

```diff
@@ -21,26 +21,26 @@ class UserDetailController extends BaseDetailController {
   }
 
   async post() {
-    const { name, email, password } = this.request.body;
+    const { name, username, password, phone } = this.request.body;
```

Comment:
Let's remove this post handler. Public routes will only support login using phone number.

Reply:
Even still, the user must be signed up first before logging in through OTP.